### DISCUSSION TITLE
Disable wrapping

### DISCUSSION THREAD

**Discussion by Lombra** [orchidjs/tom-select#563] [Ideas]
Hi,

I have a single select where the text box wraps when the text is of a certain length. I would like to be able to disable this behaviour.

![bild](https://user-images.githubusercontent.com/8340020/216759308-24c3be9b-0f0f-4cc6-bd1e-adbc9f1e2515.png)

**Comment by dantoniodi** [comment]
Hey buddy, sorry for the big delay but if it's still useful, for you or anyone struggling, here's one workaround:

```
const ts = new TomSelect(element,options);
let value;
ts.on('item_add', function() { //Closes the dropdown after an option is chosen
    value = ts.getValue();
    ts.blur();
});
ts.on('dropdown_open', function() { //Removes the selected option to avoid line wrapping
    ts.setValue(null);
});
ts.on('dropdown_close', function() { //Returns the previous value if nothing was selected
    ts.setValue(value);
});
```
Basically, just use their own [events](https://tom-select.js.org/docs/events/) to change any behavior.

**Reply by jclusso** [reply]
This works great!
